### PR TITLE
add all metrics from 'redis info' persistence section

### PR DESCRIPTION
**Pull request by zakharovvi**
Hello @oliver006 !

Thank you for this project. We've been using redis_exporter for over a year and it helps a lot.

I want to use these metrics in alerts rules:
* rdb_last_save_time
* rdb_last_bgsave_status
* aof_last_bgrewrite_status
* aof_last_write_status

Also I added other missed "persistence" metrics for consistency:
* rdb_bgsave_in_progress
* rdb_last_cow_size
* aof_last_cow_size

### CODE REVIEW

**Comment by oliver006** [comment]
Thanks for the PR!

File: exporter/redis.go [hunk 1]
```diff
@@ -81,13 +81,20 @@ var (
 
 		// # Persistence
 		"rdb_changes_since_last_save":  "rdb_changes_since_last_save",
+		"rdb_bgsave_in_progress":       "rdb_bgsave_in_progress",
+		"rdb_last_save_time":           "rdb_last_save_time",
+		"rdb_last_bgsave_status":       "rdb_last_bgsave_status",
 		"rdb_last_bgsave_time_sec":     "rdb_last_bgsave_duration_sec",
 		"rdb_current_bgsave_time_sec":  "rdb_current_bgsave_duration_sec",
+		"rdb_last_cow_size":            "rdb_last_cow_size",
```
Comment (oliver006) [inline exporter/redis.go]: Is this in bytes? 
In that case, should this be `rdb_last_cow_size_bytes` ?

File: exporter/redis.go [hunk 2]
```diff
@@ -81,13 +81,20 @@ var (
...
 		"aof_enabled":                  "aof_enabled",
 		"aof_rewrite_in_progress":      "aof_rewrite_in_progress",
 		"aof_rewrite_scheduled":        "aof_rewrite_scheduled",
 		"aof_last_rewrite_time_sec":    "aof_last_rewrite_duration_sec",
 		"aof_current_rewrite_time_sec": "aof_current_rewrite_duration_sec",
+		"aof_last_bgrewrite_status":    "aof_last_bgrewrite_status",
+		"aof_last_write_status":        "aof_last_write_status",
+		"aof_last_cow_size":            "aof_last_cow_size",
```
Comment (oliver006) [inline exporter/redis.go]: same

File: exporter/redis.go (hunk 1 above)
Comment (zakharovvi) [reply exporter/redis.go]: Yes, this in bytes:

https://redis.io/commands/info
```
rdb_last_cow_size: The size in bytes of copy-on-write allocations during the last RBD save operation
```

I renamed metric.

File: exporter/redis.go (hunk 2 above)
Comment (zakharovvi) [reply exporter/redis.go]: renamed `aof_last_cow_size` too

**Comment by zakharovvi** [comment]
@oliver006 thanks for the review!

I added `_bytes` suffix to COW metrics.

Also I renamed `rdb_last_save_time` to `rdb_last_save_timestamp_seconds`. New name follows prometheus naming convention, e.g. `prometheus_config_last_reload_success_timestamp_seconds`

**Comment by oliver006** [comment]
Released as https://github.com/oliver006/redis_exporter/releases/tag/v0.21.2

**Comment by Supermathie** [comment]
@oliver006 looks like this commit (and all of v0.21.0?) fell out of master somehow - it's missing from v.0.23.0

**Comment by oliver006** [comment]
That's very odd, let me check what's going on.

**Comment by Supermathie** [comment]
I just tested out v0.21.2 and discovered that it doesn't actually work as (I) expected… when a problem occurs the metric just **disappears** instead of changing to `0`:

\*triggers the problem*

```text
127.0.0.1:6379> info persistence
…
rdb_last_bgsave_status:err

supermathie@redis-replica: ~ $ curl -s 172.17.0.5:9121/metrics | grep bgsave_status
```

\*fixes problem*

```text
rdb_last_bgsave_status:ok

supermathie@redis-replica: ~ $ curl -s 172.17.0.5:9121/metrics | grep bgsave_status
# HELP redis_rdb_last_bgsave_status rdb_last_bgsave_statusmetric
# TYPE redis_rdb_last_bgsave_status gauge
redis_rdb_last_bgsave_status{addr="127.0.0.1:6379",alias=""} 1
```

**Pull request merged**